tests/simd/arrays.ml

```diff
@@ -538,6 +566,73 @@ module Float_arrays = struct
     fail [|#0.0|] 1;
     fail [|#0.0|] (-1)
   ;;
+
+  let () =
+    let unboxed_float32_array = unboxed_float32_array () in
+    let _0123 = f32x4 0.0s 1.0s 2.0s 3.0s in
```

Comment:
This variable name offends my delicate sensibilities.  As a general rule, reserve the `_` prefix for variables that are unused.  This can be `f32x4_0123`, or any other name of your choice.

Reply:
Sure, renamed